### PR TITLE
Feat/featured link component

### PR DESCRIPTION
**Pull request by stanleypliu**
Replica of the home hero apart from the background being a partially transparent colour (black) rather than being a gradient. 

Screenshot 2021-03-01 at 16 57 40

### CODE REVIEW

File: content/themes/ea/blocks/featured-link/block.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+<div class="featured-link<?php if ($dark_text) echo ' featured-link--dark'; ?>">
+  <div class="featured-link__body" <?php if ($background_colour) echo 'style="background-color: ' . $background_colour . ';"'; ?>>
+    <div class="featured-link__content">
+      <?php if ($title !== '') : ?>
+        <h2 class="featured-link__title"><?php echo $title; ?></h2>
+      <?php endif; ?>
+
+      <?php if ($link_url !== '') : ?>
+        <a href="<?php echo $link_url; ?>" class="featured-link__link<?php if ($link_url_external) echo ' featured-link__link--external'; ?>"<?php if ($link_url_external) echo ' target="_blank"'; ?><?php echo 'title="' . $link_text . '"'; ?>><?php echo $link_text; ?>
```
Comment (nym1k) [inline content/themes/ea/blocks/featured-link/block.php:32]: New line before echoing out the $link_text

File: content/themes/ea/blocks/featured-link/block.scss [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    @include break-min($lg) {
+      min-height: 800px;
+    }
+  }
+}
+
+.featured-link__content {
+  padding: $spacer-medium $item-padding;
+  width: 100%;
+  color: #fff;
```
Comment (nym1k) [inline content/themes/ea/blocks/featured-link/block.scss:36]: Color goes before padding

Comment (nym1k) [reply content/themes/ea/blocks/featured-link/block.scss:36]: Also, please use $color-white here

**Review by nym1k — CHANGES_REQUESTED** [review]

File: content/themes/ea/blocks/featured-link/block.php (hunk 1 above)
Comment (stanleypliu) [reply content/themes/ea/blocks/featured-link/block.php:32]: OK, I've also made sure to do this for the home hero because that has the exact same layout

**Pull request merged**